jso/apis/src/main/java/org/teavm/jso/dom/xml/DOMTokenList.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright 2017 Colin Alworth.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.teavm.jso.dom.xml;
+
+import org.teavm.jso.core.JSArrayReader;
+import org.teavm.jso.core.JSString;
+
+public interface DOMTokenList extends JSArrayReader<JSString> {
+    boolean contains(String token);
+
+    void add(String... tokens);
```

Comment:
I'm not sure if varargs are supported in JSO. Please check. If they aren't, let's think how we can work-around this. And don't forget to create an issue to support vararg.

Reply:
A good question - I'll verify more thoroughly, but I can verify that calling `add(String...)` with a single argument does work as expected presently.